contracts/utils/network.js

```diff
@@ -0,0 +1,27 @@
+const { network } = require("hardhat");
+
+const Networks = {
+  GwDevnet_v1: 'gw_devnet_v1',
+  GwTestnet_v1: 'gw_testnet_v1',
+  GwAlphanet_v1: 'gw_alphanet_v1',
+  GwBetanet_v1: 'gw_betanet_v1',
```

Comment:
I think we could deprecate gw_betanet_v1 now.

Reply:
Okay, do you think we should use `isGwMainnetV1()` or use `isNetwork(Networks.GwMainnet_v1)` directly?
Or should we have functions like:
```
runIf(isNetwork(Networks.GwMainnet_v1), () => {
  describe(...);
});
```

Would it be matter?

Reply:
`isGwMainnetV1()` LGTM.